gpii/node_modules/matchMakerFramework/src/MatchMakerUtilities.js

```diff
@@ -461,4 +461,54 @@ var fluid = fluid || require("infusion"),
             }
         });
     };
+
+    /*
+     * Converts application specific settings into common terms, keyed by application ids. Given a
+     * set of preferences, for each application block in those preferences, the settings for which
+     * inverse transformations exist will be transformed into common terms. All the common terms
+     * that have successfully been inferred from an application will be written to a block keyed by
+     * that applications ID.
+     *
+     * @preferences (Object) - An NP set (including contexts, etc)
```

Comment:
Could you update these and any others you can find to our "standard" Javer-inspired syntax which reads
@param preferences {Object} 
etc.